lib/routes.js

```diff
@@ -9,6 +9,14 @@ module.exports = {
     inputs: ['query', 'cookies'],
     sinks: sinks.cmdInjection
   },
+  cmdInjectionSemanticChainedCommands: {
+    base: '/cmdInjectionSemanticChainedCommands',
+    name: 'Command Injection Semantic Chained Commands',
+    link: 'https://www.owasp.org/index.php/Command_Injection',
+    products: ['Protect'],
+    inputs: ['query'],
```

Comment:
related to my comment above, setting the input type as `query` is just to make it work with the machinary of test-bench-utils.  we don't actually pass a query string and the command that is used is hardcoded.  This is to make sure this test doesn't trigger other protect rules (i.e. command backdoor)